src/pages/screen-2.tsx

```diff
@@ -156,51 +189,107 @@ export class Screen2 extends React.Component<Screen2Props, Screen2State> {
           <H2>Results</H2>
           {this.state.userWEP === true ? (
             <WarningBox>
-              <label>Based on the information you provided,
-                your benefits are affected by the Windfall Elimination Provision.
-                 The Windfall Elimination Provision is a Social Security rule that reduces retirement benefits for retirees
-                  with access to a pension based on non-covered employment. Click Benefit Formula at left to read more.
+              <label>
+                Based on the information you provided, your benefits are
+                affected by the Windfall Elimination Provision. The Windfall
+                Elimination Provision is a Social Security rule that reduces
+                retirement benefits for retirees with access to a pension based
+                on non-covered employment. Click Benefit Formula at left to read
+                more.
               </label>
-            </WarningBox>): null
-          }
+            </WarningBox>
+          ) : null}
           {this.state.error || !userProfile ? (
-            <WarningBox><label>Please go back and fill out all information to calculate results. </label></WarningBox>
+            <WarningBox>
+              <label>
+                Please go back and fill out all information to calculate
+                results.{" "}
+              </label>
+            </WarningBox>
           ) : (
-            <Flex>
-              <Text>Based on the information you provided, your retirement benefits will be calculated by Social Security as follows: </Text>
-              <MonthlyBenefit text={'Full Retirement Age'} number={userProfile["MPB"]} />
-              {this.state.testAge ?
-                <><Text>However, Social Security changes your monthly benefit amount if you begin to claim benefits before or after your full retirement age.
-                Use the slider below to see how your planned date of retirement will affect your monthly benefit amount.
-                      </Text>
-                  <AgeSlider 
-                    age={this.state.testAge} 
-                    handleChange={this.handleRetireChange}
-                    fullRetirementAge={fullRetirementAge ?? undefined}
+              <Flex>
+
+                <QuestionText>
+                  Which calculator would you like to see results of?
+                  </QuestionText>
+
+                <AnswerBox>
+                  <RadioButton
+                    type="radio"
+                    name="preferPiaUserCalc"
+                    value="true"
+                    onChange={ () => setPreferPiaUserCalc(true)}
```

Comment:
I see that `handleRetireChange` only updates the bottom number, not both numbers. We will need to call the full `performCalc()` to update the whole page again. Not 100% sure this will work without a deeper refactoring but worth a try.

```suggestion
                    onChange={ () => { 			setPreferPiaUserCalc(true);
                     performCalc();
                     }
                    }
```